GUI/QtQuick/qml/Send/SimpleDetails.qml

```diff
@@ -125,14 +126,48 @@ ColumnLayout  {
 
         //aliases
         title_text: qsTr("Amount")
-        input_text: "0"
-
-        input_validator: DoubleValidator{
-            bottom: 0
-            top: (from_wallet_combo.currentIndex >= 0) ?
-                     parseFloat(getWalletData(from_wallet_combo.currentIndex, WalletBalance.TotalRole)) : 0
-            notation :DoubleValidator.StandardNotation
-            locale: "en_US"
+
+//                input_validator: DoubleValidator{
+//                    bottom: 0
+//                    top: (from_wallet_combo.currentIndex >= 0) ?
+//                             parseFloat(getWalletData(from_wallet_combo.currentIndex, WalletBalance.TotalRole)) : 0
+//                    notation :DoubleValidator.StandardNotation
+//                }
+
+        //visual studio crashes when there is input_validator
+        //and we change text inside of onTextEdited
+        //it is why I have realized my validator inside of onTextEdited
+        property string prev_text : ""
+        onTextEdited : {
+
+            amount_input.input_text = amount_input.input_text.replace(",", ".")
+
+            if (amount_input.input_text.startsWith("0")
+                && !amount_input.input_text.startsWith("0.")
+                && amount_input.input_text.length > 1)
+            {
+                amount_input.input_text = "0."
+                        + amount_input.input_text.substring(1, amount_input.input_text.length)
+            }
+            try {
+                var input_number =  Number.fromLocaleString(Qt.locale("en_US"), amount_input.input_text)
```

Comment:
Wasn't it easier to get `input_number` as `parseFloat(amount_input.input_text)`? You're doing the same below when passing data to bsApp method. We should use the same conversion function in all places, IMO.

Reply:
No parseFloat does not return error for example in this case "0.hghtt", this function check only 1 digit in number https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat#return_value When we are sure that string is double we can use parseFloat, because it is more simple, when we not sure, Number is better variant

Reply:
It is non-standard place, I have realized my own validator...so in other places in 99% will be used parseFloat - it is more simple. So only reason to use Number, because this function generate exceptions every time when argument is not double

Reply:
Hmm...I have not found comparison between parseFloat and Number.fromLocaleString. But I beleive that parseFloat is faster...To qml it is important. So I suggest use parseFloat  in all places where we sure that string is number, in other places use Number.fromLocaleString (more powerfull but slower function).